### PR TITLE
{tools}[foss/2021a,foss/2021b] fastparquet v0.7.2, fastparquet v0.8.0 w/ Python 3.9.5 + 3.9.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node315.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b45a7becd00925a82224205f16479840 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node148.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/754c093fae989a0240dfd1df77148268 for a full test report.

File: easybuild/easyconfigs/f/fastparquet/fastparquet-0.7.2-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'fastparquet'
+version = '0.7.2'
+
+homepage = "https://fastparquet.readthedocs.io/"
+description = """fastparquet is a python implementation of the parquet format, aiming integrate
```
Comment (verdurin) [inline easybuild/easyconfigs/f/fastparquet/fastparquet-0.7.2-foss-2021a.eb:7]:
```suggestion
description = """fastparquet is a python implementation of the parquet format, aiming to integrate
```

File: easybuild/easyconfigs/f/fastparquet/fastparquet-0.8.0-foss-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'fastparquet'
+version = '0.8.0'
+
+homepage = "https://fastparquet.readthedocs.io/"
+description = """fastparquet is a python implementation of the parquet format, aiming integrate
```
Comment (verdurin) [inline easybuild/easyconfigs/f/fastparquet/fastparquet-0.8.0-foss-2021b.eb:7]:
```suggestion
description = """fastparquet is a python implementation of the parquet format, aiming to integrate
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/8b358d7a2f47d2be332f4b3225b2abf8 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/36219c1c09738abe0efe7dfed37c6b71 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**